demosplan/DemosPlanCoreBundle/Logic/LinkMessageSerializable.php

```diff
@@ -49,8 +49,6 @@ public function __construct(
      * @param string $routeName
      * @param array  $routeParameters
      * @param string $linkText
-     *
-     * @return LinkMessageSerializable
```

Comment:
I guess the integration branch must be merged into this branch f_T30974_continue_with_color_brand to get a clean diff.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/876/commits/1ac46ddf6e57dac646ff3d7197cf9ef6fde5334f